fuzz/src/chanmon_consistency.rs

```diff
@@ -516,6 +506,10 @@ pub fn do_test(data: &[u8]) {
 							// Can be generated due to a payment forward being rejected due to a
 							// channel having previously failed a monitor update
 						},
+						events::MessageSendEvent::HandleError { .. } => {
```

Comment:
This really needs to check that the error in question is an IgnoreError action, and panic otherwise.